package.json

```diff
@@ -20,14 +17,16 @@
     "prebuild": "yarn clean",
     "type-check": "tsc",
     "dev": "npm run type-check -- --watch",
-    "build": "npm run build:types && npm run build:esm && npm run build:cjs",
+    "build": "npm run build:types && npm run build:rollup",
+    "build:rollup": "rollup --c ./config/rollup.config.js",
     "build:types": "tsc --emitDeclarationOnly",
-    "build:esm": "babel src --out-dir dist --extensions '.ts,.tsx' --ignore 'src/**/*.stories.tsx','src/**/*.test.tsx'",
-    "build:cjs": "BABEL_ENV=cjs babel src --out-dir dist/cjs --extensions '.ts,.tsx' --ignore 'src/**/*.stories.tsx','src/**/*.test.tsx'",
+    "postbuild": "npm run prepdist",
+    "prepdist": "node ./config/prepareDist.js",
     "test": "is-ci \"test:coverage\" \"test:watch\"",
-    "test:coverage": "NODE_ENV=test jest test --coverage",
-    "test:watch": "NODE_ENV=test jest test --watch",
+    "test:coverage": "jest test --coverage",
+    "test:watch": "jest test --watch",
     "lint": "eslint src",
+    "dev-pack": "yarn build && cd dist && npm pack",
```

Comment:
`pack` is a bit of a weird name 🤔  anyway, can we maybe describe the commands somewhere so we know what each command does and when to use it?

Reply:
I named it like this because the npm command is also `npm pack` 🤔 
used to just make a package file you can use in testing. 

I will add some documentation to the solar package readme on how to test new build etc.